src/EvoSC.Common/Database/Migrations/202210100826_AddPlayersTable.cs

```diff
@@ -16,6 +18,15 @@ public override void Up()
             .WithColumn("LastVisit").AsDateTime().Nullable()
             .WithColumn("CreatedAt").AsDateTime().WithDefault(SystemMethods.CurrentDateTime)
             .WithColumn("UpdatedAt").AsDateTime().WithDefault(SystemMethods.CurrentDateTime);
+
+        Insert.IntoTable("Players").Row(new
```

Comment:
In that case, perhaps we should also separate table creation and seeding migrations. So that its better organized, either with a naming scheme or directories/namespaces.